client/rpc/src/system/mod.rs

```diff
@@ -125,6 +152,10 @@ impl<B: traits::Block> SystemApi<B::Hash, <B::Header as HeaderT>::Number> for Sy
 	fn system_remove_reserved_peer(&self, peer: String)
 		-> Compat<BoxFuture<'static, std::result::Result<(), rpc::Error>>>
 	{
+		if let Err(err) = self.deny_unsafe.check_if_safe() {
```

Comment:
Should be using `bail_if_unsafe!` too, right?

Reply:
Oops! You're right, fixed. Thanks for spotting this!